lib/repl.js

```diff
@@ -624,7 +624,7 @@ REPLServer.prototype.complete = function(line, callback) {
           completionGroupsLoaded();
         } else {
           this.eval('.scope', this.context, 'repl', function(err, globals) {
-            if (err || !globals) {
+            if (err || !globals || !Array.isArray(globals)) {
```

Comment:
Unsure, I just wanna let `globals == '.scope'` be `true`, maybe `err || !globals || !globals.sort` is better? 

But I guess it's not the `ArrayStream`, because the process crashed here: https://github.com/yorkie/io.js/blob/fix/repl-tab/lib/repl.js#L720, so we should make sure it has `sort` function to fix it.